decidim-admin/app/views/layouts/decidim/admin/_application.html.erb

```diff
@@ -26,7 +26,7 @@
       </div>
     </div>
     <%= render partial: "layouts/decidim/admin/template_bottom" %>
-    <%= render partial: "decidim/shared/confirm_modal" %>
+    <%= render partial: "decidim/shared/admin_confirm_modal" %>
```

Comment:
If this is a partial that is being used only by admin, maybe it has more sense to be stored in any of the following paths:
- `decidim-admin/app/views/layouts/decidim/admin`
- `decidim-admin/app/views/decidim/admin/shared`

Reply:
Moved